client/src/components/tenant/Tenant.css

```diff
@@ -10,7 +10,7 @@
 .payment-history-container {
   max-height: 250px;
   overflow-y: auto;
-  margin-left: 14px;
```

Comment:
I think the styling for the profile section might still need to be adjusted. Not too sure if this change happened earlier, but the profile image is no longer centered and the `Edit` text is not visible in the button
![image](https://github.com/ryanzhxu/estate-flow/assets/100311477/a3cf31ee-09d8-4d29-ba3d-d7bc38d3b8f3)

Reply:
I guess it's not too big a deal now tho.